tests/test_tokenization_common.py

```diff
@@ -18,10 +18,31 @@
 import pickle
 import shutil
 import tempfile
+from collections import OrderedDict
+from typing import Dict, Tuple, Union
 
 from tests.utils import require_tf, require_torch
 
 
+def merge_model_tokenizer_mappings(
+    model_mapping: "Dict[PretrainedConfig, Union[PreTrainedModel, TFPreTrainedModel]]",  # noqa: F821
+    tokenizer_mapping: "Dict[PretrainedConfig, Tuple[PreTrainedTokenizer, PreTrainedTokenizerFast]]",  # noqa: F821
+) -> "Dict[Union[PreTrainedTokenizer, PreTrainedTokenizerFast], Tuple[PretrainedConfig, Union[PreTrainedModel, TFPreTrainedModel]]]":  # noqa: F821
+    configurations = list(model_mapping.keys())
+    model_tokenizer_mapping = OrderedDict([])
+
+    for configuration in configurations:
+        model = model_mapping[configuration]
+        tokenizer = tokenizer_mapping[configuration][0]
+        tokenizer_fast = tokenizer_mapping[configuration][1]
+
+        model_tokenizer_mapping.update({tokenizer: (configuration, model)})
+        if tokenizer_fast is not None:
+            model_tokenizer_mapping.update({tokenizer_fast: (configuration, model)})
+
+    return model_tokenizer_mapping
+
+
```

Comment:
Here the arguments are between quotes as they can't be imported when a framework is not available; i.e. `TFPreTrainedModel` may not be imported unless TensorFlow >= 2.0 is installed in the environment.

Reply:
- maybe you can just put the actual class names in quotes, not the Dict/Union/etc. (that's what we do elsewhere in the code, I think)
- a `PreTrainedTokenizerFast` is a `PreTrainedTokenizer` (and I don't expect that to change in the future) so you can maybe drop the first one.